.github/workflows/e2e-update-generic-server.yaml

```
@@ -0,0 +1,460 @@
+        with:
+          fail_on_failure: true
+          include_passed: true
+          detailed_summary: true
+          annotate_only: true
+          require_tests: true
+          report_paths: '**/*results.xml'
+
+      - uses: actions/upload-artifact@043fb46d1a93c77aae656e7c1c64a875d1fc6a0a # v7.0.1
+        if: always()
+        with:
+          name: ${{ matrix.os }}-generic-server-update-e2e-${{ matrix.installation }}
```

actionlint (1.7.12)

[error] property "installation" is not defined in object type {os: string}

(expression)